### PR TITLE
feat: android icons

### PR DESCRIPTION
**Pull request by admbtlr**
This PR resolves [MOPLAT-439]

### Description

image

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-439]: https://artsyproduct.atlassian.net/browse/MOPLAT-439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+  <background android:drawable="@mipmap/ic_launcher_background"/>
+  <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+  <monochrome android:drawable="@mipmap/ic_launcher_monochrome"/>
+</adaptive-icon>
```
Comment (pvinis) [inline android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:6]:
```suggestion
</adaptive-icon>

```
just a quick eof newline

File: android/build.gradle [hunk 2]
```diff
@@ -35,6 +35,16 @@ buildscript {
 
 allprojects {
     repositories {
+        exclusiveContent {
```
Comment (pvinis) [inline android/build.gradle:38]: whats all that for?

File: ios/.tool-versions [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ruby 2.7.4
```
Comment (pvinis) [inline ios/.tool-versions:1]: we have this one https://github.com/artsy/energy/blob/main/.tool-versions. if we need newer versions, better change it there instead.

File: android/build.gradle (hunk 2 above)
Comment (admbtlr) [reply android/build.gradle:38]: Yeah, weird issue that stops android compiling:
https://artsy.slack.com/archives/C02NAJ2CGLW/p1673626323228539?thread_ts=1673625946.823429&cid=C02NAJ2CGLW
https://stackoverflow.com/questions/74333132/task-react-native-async-storage-async-storagegeneratedebugrfile-failed

File: ios/.tool-versions (hunk 3 above)
Comment (admbtlr) [reply ios/.tool-versions:1]: Aah, sorry, forgot that I'd changed this while wrestling with asdf. I'll remove it.

File: android/build.gradle (hunk 2 above)
Comment (pvinis) [reply android/build.gradle:38]: oh wow. cool 👍

**Review by pvinis — APPROVED** [review]

**Pull request merged**